src/ASM/ASMs2Dmx.C

```diff
@@ -638,7 +638,7 @@ bool ASMs2Dmx::integrate (Integrand& integrand,
               utl::getGmat(Jac,dXidu,fe.G);
 
             // Cartesian coordinates of current integration point
-            X = Xnod * fe.basis(geoBasis);
+            X.assign(Xnod * fe.basis(geoBasis));
```

Comment:
There is no point in switching to using the assign method until you actually assign the X.u parameters.
So maybe delay touching this file until mixed is fully supported.

Reply:
done.